docs/source/markdown/podman-save.1.md

```diff
@@ -35,8 +35,9 @@ Write to a file, default is STDOUT
 
 #### **--format**=*format*
 
-Save image to **oci-archive, oci-dir** (directory with oci manifest type), or **docker-dir** (directory with v2s2 manifest type)
+Save image to **docker-archive**, **oci-archive** (see `containers-transports(5)`), **oci-dir** (`transport` transport), or **docker-dir** (`dir` transport with v2s2 manifest type).
```

Comment:
> `transport` transport

a typo?

Reply:
Definitely. Thanks for catching! Rushing multiple PRs at once is not my strong suit :^)